website/src/theme/SearchBar/algolia.css

```diff
@@ -0,0 +1,328 @@
+    center no-repeat;
+  background-size: 100%;
+  overflow: hidden;
+  text-indent: -9000px;
+  width: 110px;
+  height: 100%;
+  display: block;
+  margin: 0 5px 0 auto;
+}
```

Comment:
This was just pasted from Algolia or?

Reply:
When you install Algolia this file comes with it.

Reply:
I can convert this file to scss but I'm not sure if this will affect the way Algolia works and what will happen if it gets updated.